behave/run_test_local.sh

```diff
@@ -1,6 +1,15 @@
+#!/bin/bash
 export DOCKER_BUILDKIT=0
+a11y=${1:-"not_set"}
```

Comment:
```
a11y=${1:-"not_set"}
if [ $a11y = "not_set" ]; then
  export A11Y_SET="false"
else
  export A11Y_SET="true"
fi```
Is this more of the build stuff we removed?

Reply:
Removed, thanks